diff --git a/arch/arm/boot/dts/acme-acqua.dts b/arch/arm/boot/dts/acme-acqua.dts
index 5d38766..f3f4949 100644
--- a/arch/arm/boot/dts/acme-acqua.dts
+++ b/arch/arm/boot/dts/acme-acqua.dts
@@ -173,9 +173,9 @@
     panel: panel {
         /* compatible = "acme,43inch", "simple-panel"; */
         /* compatible = "acme,50inch", "simple-panel"; */
-        /* compatible = "acme,70inch", "simple-panel"; */
+        compatible = "acme,70inch", "simple-panel";
         
-        status = "disabled";
+        status = "okay";
 
         port@0 {
             panel_input: endpoint@0 {
